compiled/js/skipto.js

```diff
@@ -1,8 +1,8 @@
-/*! skipto - v4.0.1 - 2021-01-15
-* https://github.com/paypal/skipto
-* Copyright (c) 2021 PayPal Accessibility Team and University of Illinois; Licensed BSD */
- /*@cc_on @*/
-/*@if (@_jscript_version >= 5.8) @*/
+/*! skipto - v3.1.5 - 2021-01-15

```

Comment:
Something wrong here or the compiled version number would have gone down

Reply:
fixed